src/affine.jl

```diff
@@ -123,18 +193,15 @@ function qd_affine(fixed, moving, mxshift, linmins, linmaxs;
                                              SD = SD, minwidth=mw, initial_tfm=initial_tfm, thresh=thresh, print_interval=print_interval, kwargs...)
     print_interval < typemax(Int) && print("Running fine step\n")
     mw = mw./100
-    linmins, linmaxs = scalebounds(linmins, linmaxs, 0.5)
+    linmins, linmaxs = scalebounds(linmins, linmaxs, 0.5) # should these be narrowed further?
```

Comment:
What's the design logic behind using 0.5*linmins|linmax as the parameters for fine? Shouldn't the tfm from coarse get similar sized images within similar distances of the the final transformation, regardless of how far off the starting point was?

Reply:
That's my sense too. We might change a number of things re affine, including shifting to more of a `Δc` parametrization (like in `default_minrot`). But let's do that in a separate PR.